data/en/mbs_0253.json

```diff
@@ -0,0 +1,455 @@
+                                            "voluntary workers",
+                                            "former employees only receiving pension",
+                                            "self-employed workers"
+                                        ]
+                                    }
+                                ]
+                            },
+                            "type": "General"
+                        }],
+                        "title": "Employees"
+                    },
+                    {
+                        "type": "ConfirmationQuestion",
+                        "title": "Employees",
+                        "id": "confirm-zero-employees-block",
+                        "skip_conditions": [{
+                            "when": [{
+                                "id": "total-number-employees",
+                                "condition": "greater than",
+                                "value": 0
+                            }]
+                        }],
+                        "questions": [{
+                            "type": "General",
+                            "answers": [{
+                                "type": "Radio",
+                                "id": "confirm-zero-employees-answer",
+                                "q_code": "d50",
+                                "options": [{
+                                        "label": "Yes this is correct",
+                                        "value": "Yes"
+                                    },
+                                    {
+                                        "label": "No I need to change this",
+                                        "value": "No"
+                                    }
+                                ],
+                                "mandatory": true
+                            }],
+                            "id": "confirm-zero-employees-question",
+                            "title": "On {{exercise.employment_date|format_date}}, the number of employees for {{respondent.trad_as_or_ru_name}} was <em>0</em>, is this correct?"
+                        }],
+                        "routing_rules": [{
+                                "goto": {
+                                    "when": [{
+                                        "value": "No",
+                                        "id": "confirm-zero-employees-answer",
+                                        "condition": "equals"
+                                    }],
+                                    "block": "number-of-employees"
+                                }
+                            },
+                            {
+                                "goto": {
+                                    "block": "significant-change"
+                                }
+                            }
+                        ]
+                    },
+                    {
+                        "type": "Question",
```

Comment:
Question is wrong, slides are correct
Of the 5 total employees employed on 10 June 2016, please specify the number of:
slides
Of the {{employee total figure}} total employees employed on {{employment_date}}, how many male and female employees worked the following hours?